arcade/types.py

```diff
@@ -49,6 +52,22 @@
 
 RGBA255OrNormalized = Union[RGBA255, RGBANormalized]
 
+# equivalent to the following filters (under arcade.gl.enums):
+# NEAREST, LINEAR, NEAREST_MIPMAP_NEAREST, LINEAR_MIPMAP_NEAREST,
+# NEAREST_MIPMAP_LINEAR, LINEAR_MIPMAP_LINEAR
+OpenGlFilters: TypeAlias = Union[Literal[9728], Literal[9729], Literal[9984],
+                                 Literal[9985], Literal[9986], Literal[9987]]
+
+# equivalent to the following blend_functions (under arcade.gl.enums):
+# ZERO, ONE, SRC_COLOR, ONE_MINUS_SRC_COLOR,
+# SRC_ALPHA, ONE_MINUS_SRC_ALPHA, DST_ALPHA,
+# ONE_MINUS_DST_ALPHA, DST_COLOR, ONE_MINUS_DST_COLOR
+BlendFunctions: TypeAlias = Union[Literal[0], Literal[1], Literal[768],
+                                  Literal[769], Literal[770], Literal[771],
+                                  Literal[772], Literal[773], Literal[774],
+                                  Literal[775]]
+
+
```

Comment:
I already asked to make some of the members in `enums.py` actual enum classes but the response was negative so idk

Reply:
> but the response was negative
That's disappointing.  Do you remember where that discussion took place, anything about the context, or the reasons given for avoiding it? I was hoping to explore moving to `IntEnum` in the long term once more pressing matters are settled.

Reply:
tho i feel like that it's not a definitive answer https://discord.com/channels/458662222697070613/458662458198982676/1137813484344721489

Reply:
Let's not invest too much energy here.  If we use `PyGlEnum`, it's less contentious and we can get this merged.

https://github.com/pythonarcade/arcade/blob/09832834846cae44068e752ff0d96f80c28f9034/arcade/gl/types.py#L12-L13

Using these Unions of literals is *technically* more strict, but I don't think it's practical.  It will cause more friction than benefit.  It also add repetition.